src/Forms/FormField.php

```diff
@@ -333,7 +333,7 @@ public static function create_tag($tag, $attributes, $content = null)
             }
         }
 
-        if ($content || $tag != 'input') {
+        if ($content || !in_array($tag, array('input', 'meta'))) {
```

Comment:
Can we add all the void elements to this array? see http://w3c.github.io/html/syntax.html#void-elements